src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -39,8 +44,11 @@
 
     @Test
     public void getStatisticAboutOrange() {
+        try {
         workWithFile.getStatistic("orange.csv", ORANGE_RESULT_FILE);
-
+        } catch (FileNotFoundException e) {
```

Comment:
Catching `FileNotFoundException` is not appropriate here because the `getStatistic` method can throw `IOException`. Consider catching `IOException` instead.